src/components/input/input.e2e.ts

```diff
@@ -202,189 +202,386 @@ describe("calcite-input", () => {
-    await numberHorizontalItemUp.click();
-    await page.waitForChanges();
-    expect(await element.getProperty("value")).toBe("6");
-  });
+    it("correctly stops decrementing value when min is set", async () => {
+      await page.setContent(html` <calcite-input type="number" min="10" value="11"></calcite-input> `);
 
-  it("correctly stops decrementing value when min is set", async () => {
-    const page = await newE2EPage();
-    await page.setContent(html` <calcite-input type="number" min="10" value="11"></calcite-input> `);
+      const element = await page.find("calcite-input");
+      const numberHorizontalItemDown = await page.find("calcite-input >>> .number-button-item[data-adjustment='down']");
 
-    const element = await page.find("calcite-input");
-    const numberHorizontalItemDown = await page.find("calcite-input >>> .number-button-item[data-adjustment='down']");
+      await numberHorizontalItemDown.click();
+      await page.waitForChanges();
+      expect(await element.getProperty("value")).toBe("10");
+      await numberHorizontalItemDown.click();
+      await page.waitForChanges();
+      expect(await element.getProperty("value")).toBe("10");
+    });
 
-    await numberHorizontalItemDown.click();
-    await page.waitForChanges();
-    expect(await element.getProperty("value")).toBe("10");
-    await numberHorizontalItemDown.click();
-    await page.waitForChanges();
-    expect(await element.getProperty("value")).toBe("10");
-  });
+    it("correctly stops incrementing value when max is set", async () => {
+      await page.setContent(html` <calcite-input type="number" max="10" value="9"></calcite-input> `);
 
-  it("correctly stops incrementing value when max is set", async () => {
-    const page = await newE2EPage();
-    await page.setContent(html` <calcite-input type="number" max="10" value="9"></calcite-input> `);
+      const element = await page.find("calcite-input");
+      const numberHorizontalItemUp = await page.find("calcite-input >>> .number-button-item[data-adjustment='up']");
 
-    const element = await page.find("calcite-input");
-    const numberHorizontalItemUp = await page.find("calcite-input >>> .number-button-item[data-adjustment='up']");
+      await numberHorizontalItemUp.click();
+      await page.waitForChanges();
+      expect(await element.getProperty("value")).toBe("10");
+      await numberHorizontalItemUp.click();
+      await page.waitForChanges();
+      expect(await element.getProperty("value")).toBe("10");
+    });
 
-    await numberHorizontalItemUp.click();
-    await page.waitForChanges();
-    expect(await element.getProperty("value")).toBe("10");
-    await numberHorizontalItemUp.click();
-    await page.waitForChanges();
-    expect(await element.getProperty("value")).toBe("10");
+    it("should emit event when up or down clicked on input", async () => {
+      await page.setContent(html` <calcite-input type="number" max="0" value="-2"></calcite-input> `);
```

Comment:
Nitpick: remove leading and trailing space between HTML and helper. This applies to other snippets.